.travis.yml

```diff
@@ -12,15 +12,18 @@ matrix:
         - python: "3.5"
         - python: "3.6"
         - python: "3.7-dev"
-        - env: BUILD_DOCKER=1
+        - env:
+            - BUILD_DOCKER=1
+            - DOCKER_USERNAME=sbrunner
+            - DOCKER_PASSWORD=<replace by `travis encrypt DOCKER_PASSWORD=real-password`>
```

Comment:
I am not sure it's such a good idea to have a docker_password (even encrypted) in a visible repository visible file...

Reply:
https://docs.travis-ci.com/user/encryption-keys/

```$ travis encrypt --com DOCKER_PASSWORD="secretvalue"```

Reply:
ok I guess it's a non issue then...